src/main/settings.py

```diff
@@ -35,6 +35,7 @@
 ALLOWED_HOSTS: list[str] = []
 
 INSTALLED_APPS = [
+    "unfold",
```

Comment:
Nitpick: I like to add extra installed apps at the end of the array because it gives a sense of the order in which things were added.

Reply:
Understandable! For this specifically, the [installation documentation](https://github.com/unfoldadmin/django-unfold?tab=readme-ov-file#installation) lists that "unfold" needs to be listed before "django.contrib.admin", which is why I placed it at the beginning instead of the end. Otherwise, I agree and would have placed it at the end.

Reply:
Ah, good to know. That makes sense.